### PR TITLE
post-1.18.4 - update changelog and version in 1.18.x

### PR DESCRIPTION
**Pull request by jmurret**
### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

**Review by NicoletaPopoviciu — APPROVED** [review]
thank you for capturing the 1.18.3 changes as well; looks like the version update was missed during the last patch release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+## 1.18.4 Enterprise (August 26, 2024)
+
+Enterprise LTS: Consul Enterprise 1.18 is a Long-Term Support (LTS) release.
+
+SECURITY:
```
Comment (zalimeni) [inline CHANGELOG.md:5]:
```suggestion
SECURITY:

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+* ui: Upgrade modules with d3-color as a dependency to address denial of service issue in d3-color < 3.1.0
+
+IMPROVEMENTS:
+
+* Use Envoy's default for a route's validate_clusters option, which is false. This fixes a case where non-existent clusters could cause a route to no longer route to any of its backends, including existing ones. [[GH-21587](https://github.com/hashicorp/consul/issues/21587)]
+
+## 1.18.3 Enterprise (July 11, 2024)
```
Comment (zalimeni) [inline CHANGELOG.md:12]:
```suggestion
## 1.18.3 Enterprise (July 11, 2024)

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+**Enterprise LTS**: Consul Enterprise 1.18 is a Long-Term Support (LTS) release.
+SECURITY:
```
Comment (zalimeni) [inline CHANGELOG.md:14]:
```suggestion

SECURITY:

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+* Upgrade envoy module dependencies to version 1.27.7, 1.28.5 and 1.29.7 or higher to resolve [CVE-2024-39305](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-39305) [[GH-21524](https://github.com/hashicorp/consul/issues/21524)]
+* Upgrade go version to 1.22.5 to address [CVE-2024-24791](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-24791) [[GH-21507](https://github.com/hashicorp/consul/issues/21507)]
+* Upgrade go-retryablehttp to address [CVE-2024-6104](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6104) [[GH-21384](https://github.com/hashicorp/consul/issues/21384)]
+* agent: removed reflected cross-site scripting vulnerability [[GH-21342](https://github.com/hashicorp/consul/issues/21342)]
+* ui: Pin and namespace sub-module dependencies related to the Consul UI [[GH-21378](https://github.com/hashicorp/consul/issues/21378)]
+  IMPROVEMENTS:
```
Comment (zalimeni) [inline CHANGELOG.md:20]:
```suggestion

IMPROVEMENTS:

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
+* mesh: update supported envoy version 1.29.4
+* mesh: update supported envoy version 1.29.5 in addition to 1.28.4, 1.27.6. [[GH-21277](https://github.com/hashicorp/consul/issues/21277)]
+* upgrade go version to v1.22.3. [[GH-21113](https://github.com/hashicorp/consul/issues/21113)]
+* upgrade go version to v1.22.4. [[GH-21265](https://github.com/hashicorp/consul/issues/21265)]
+  BUG FIXES:
```
Comment (zalimeni) [inline CHANGELOG.md:25]:
```suggestion

BUG FIXES:

```

File: version/VERSION [hunk 6]
```diff
@@ -1 +1 @@
-1.18.3-dev
+1.18.5-dev
```
Comment (zalimeni) [inline version/VERSION:1]: Oof.

**Review by zalimeni — APPROVED** [review]
Whitespace edits, otherwise 👍🏻

**Pull request merged**